src/salaryAndBenefits/SalaryAndBenefits.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from "./salaryAndBenefits.module.css";
+import Text from "src/components/text/Text";
+import {SalaryAndBenefits as SalaryAndBenefitsPayload} from "studio/lib/payloads/salaryAndBenefits";
+
+interface SalaryAndBenefitsProps {
+  salaryAndBenefits: SalaryAndBenefitsPayload
+}
+
+export default function SalaryAndBenefits({salaryAndBenefits}: SalaryAndBenefitsProps) {
```

Comment:
To keep the flexibility as well as the code conventions of the repo: 

```suggestion
const SalaryAndBenefits = ({salaryAndBenefits}: SalaryAndBenefitsProps) => {
```

and `export default SalaryAndBenefits`

Reply:
Just following [Mikael](https://blog.variant.no/a-better-way-to-type-react-components-9a6460a1d4b7), but consistency is nice as well 🤗